ansible/roles/test/tasks/copp.yml

```diff
@@ -28,6 +25,36 @@
       copy: src=roles/test/files/ptftests dest=/root
       delegate_to: "{{ ptf_host }}"
 
+# Add case for Force10-S6100 since this platform has 64 ports
+# TODO: Generalize the code so that it fits for any topologies
+    - set_fact:
+        test_port_index: "3"
+        test_port_src_ip: "10.0.0.7"
+        test_port_peer_ip: "10.0.0.6"
+        test_port_alias: "Ethernet12"
+      when: minigraph_hwsku != 'Force10-S6100'
+
+    - set_fact:
+        test_port_index: "1"
+        test_port_src_ip: "10.0.0.1"
+        test_port_peer_ip: "10.0.0.0"
+        test_port_alias: "Ethernet1"
+      when: minigraph_hwsku == 'Force10-S6100'
+
+    - block:
+      - name: Stop the original ptf_nn_agent thread started by supervisord
+        shell: supervisorctl stop ptf_nn_agent
+        vars:
+          ansible_shell_type: docker
+          ansible_python_interpreter: docker exec -i syncd python
```

Comment:
Why do we need this?
Should we restore it back after we finished with the test?